initializer/main.go

```diff
@@ -164,6 +166,20 @@ func run(cmd *cobra.Command, _ []string) (retErr error) {
 		}
 	}
 
-	log.Info("Initializer done")
-	return nil
+	cryptsetupDevicePath := os.Getenv("CRYPTSETUP_DEVICE")
+	if cryptsetupDevicePath == "" {
+		log.Info("Initializer done")
+		return nil
+	}
+
+	log.Info("Setting up encrypted mount")
+
+	flags := &cryptsetupFlags{
+		devicePath:       cryptsetupDevicePath,
+		volumeMountPoint: "/state",
+	}
+
+	ctx, _ = signal.NotifyContext(ctx, syscall.SIGTERM, syscall.SIGINT)
```

Comment:
```suggestion
	ctx, cancel := signal.NotifyContext(ctx, syscall.SIGTERM, syscall.SIGINT)
	defer cancel()
```
I think you're supposed to defer that function, although it probably doesn't matter a lot considering that we're exiting anyway.